api-platform/core/CVE-2025-31481.yaml

```diff
@@ -0,0 +1,14 @@
+title:     "GraphQL query operations security can be bypassed"
+link:      https://github.com/advisories/GHSA-cg3c-245w-728m
+cve:       CVE-2025-31481
+branches:
+    '3.4':
+        time:     2025-04-03 15:02:00
+        versions: ['>=3.4.0', '<3.4.17']
```

Comment:
the GitHub advisory says `<3.4.17` is affected, so I would remove the lower bound here